packages/babel-plugin-transform-block-scoping/test/fixtures/tdz/exported-fn/output.mjs

```diff
@@ -4,7 +4,7 @@ import { foo } from "somewhere";
 // foo might call "bar"
 foo();
 babelHelpers.tdz("a");
-a = void 0;
+var a = void 0;
```

Comment:
The plugin now always keeps `var`, because we cannot safely remove it anyway. For example:
```js
maybeCall(() => a);
let { a, b } = c;
```
becomes
```js
var a = babelHelpers.temporalUndefined;
maybeCall(() => _tdzCheck(a));
var { a, b } = c;
```

We can easily remove `var` from `var a = babelHelpers.temporalUndefined`; I didn't do it in this PR to avoid diffs in every file.

Reply:
I agree with this particular scenario: it's safer when range tracking isn't 100% reliable.